drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -129,6 +129,24 @@ class DragAndDropBlock(
         enforce_type=True,
     )
 
+    item_sizing = String(
+        display_name=_("Item sizing"),
+        help=_(
+            "Standard sizing: The item will snap to a rectangular size for "
+            "items with a small amount of text, or double in size to a square "
+            "for items containing an image or large amounts of text. "
+            "Free sizing: The item size will fit the specified maximum width, "
+            "or freely expand with the size of the content."
+        ),
+        scope=Scope.settings,
+        values=[
+            {"display_name": _("Standard sizing"), "value": Constants.STANDARD_SIZING},
+            {"display_name": _("Free sizing"), "value": Constants.FREE_SIZING},
+        ],
+        default=Constants.STANDARD_SIZING,
```

Comment:
Could you remove this default value? Setting this as the default means that all old blocks that don't have this setting will automatically use standard sizing and this might break their layout. If there is no default then the old behaviour (Free Sizing) can be used when no value is present.

Reply:
How about we set free sizing as default to stay backward compatible, also rename standard sizing to fixed sizing because the name `standard` implies default?

Reply:
Setting it as the default will also set it as the default in the UI. I'm not sure about calling it Fixed sizing since it can toggle between a square and rectangle if we were to go that way perhaps it could be called Automatic Sizing? I'll post on the ticket.

Reply:
`Setting it as the default will also set it as the default in the UI`, It would still be default in UI If we don't set it as default because if there is no default we will still be setting free sizing in UI to stay backward compatible.

Reply:
My point was that we wouldn't want it as the default in the UI. Right now Standard Sizing is the first option so it's kind of selected by default even if it isn't explicitly set.